MonoGame.Framework/Graphics/GraphicsDevice.cs

```diff
@@ -175,6 +175,13 @@ public GraphicsAdapter Adapter
         /// The metrics are reset every frame after draw within <see cref="GraphicsDevice.Present"/>. 
         /// </summary>
         public GraphicsMetrics Metrics { get { return _graphicsMetrics; } set { _graphicsMetrics = value; } }
+
+        private GraphicsDebug _graphicsDebug;
+
+        /// <summary>
+        /// Access debugging APIs for the graphics subsystem.
+        /// </summary>
+        public GraphicsDebug GraphicsDebug { get { return _graphicsDebug; } set { _graphicsDebug = value; } }
```

Comment:
> is there a reason why GraphicsMetrics has a public setter?

We wanted the user to be able to copy the metric data, reset it, then restore the old copy later.  This would let you measure metrics of individual subsystems while not loosing the metrics before the system ran.